pkg/runtime/cli.go

```diff
@@ -295,6 +297,64 @@ func FromFlags() (*DaprRuntime, error) {
 		healthThreshold = int32(*appHealthThreshold)
 	}
 
+	var certChain *credentials.CertChain
+
+	// Config and resiliency need the operator client
+	var operatorClient operatorV1.OperatorClient
+	if *mode == string(modes.KubernetesMode) {
+		log.Info("Initializing the operator client")
+		certChain, err = security.GetCertChain()
+		if err != nil {
+			return nil, err
+		}
+		client, conn, clientErr := client.GetOperatorClient(context.TODO(), *controlPlaneAddress, security.TLSServerName, certChain)
```

Comment:
Shouldn't this context have a timeout - even a long one like 1 minute

Reply:
Timeout has been set to 30 seconds inside the `GetOperatorClient` function to avoid blocking the app. So it is safe to leave context todo here, I'll add a comment for this.

Reply:
In that case please use `context.Background()` or any parent context that may be defined in the code.

`context.TODO()` is meant to be a ... TODO to be fixed in the future.

Reply:
Ok to leave as TODO?

this will make it easier to pick up when we finish daprd graceful shutdown

Reply:
I am ok with both `context.Background()` and `context.TODO()`, since the original code uses `context.TODO()` and with JoshVanL's comment, I am inclined to leave it as `context.TODO()` here.